setup.py

```diff
@@ -34,20 +34,23 @@
     u'your raw data with rich annotations, comments, tags and stars.')
 
 def check_before_upload():
+    """Raise UserWarning if frontend build is not present or is not recent,
```

Comment:
Let's have the docstring consistent with the rest, i.e. Google style:

"""Warn user if frontend build is not present or is not recent.

Make sure that .js and .css bundles included in the PyPI package are up to date.

Raises:
    UserWarning
"""